ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -224,11 +224,20 @@ RUN git clone https://github.com/c9/core.git /home/user/.c9sdk \
 ADD workspace-janitor.js /home/user/.c9sdk/configs/ide/
 RUN sudo chown user:user /home/user/.c9sdk/configs/ide/workspace-janitor.js
 
+# Install the Theia IDE with all features available.
+COPY --chown=user theia /home/user/.theia/
```

Comment:
Wait, there is a `--chown` option on `COPY`? This is amazing! There are so many places where we `ADD file` or `COPY file` and then `RUN sudo chown user:user file`!

Reply:
Yes, though it's added recently so keep an eye on your Docker version.